### PR TITLE
Recent packages RSS feed

### PR DESCRIPTION
**Pull request by finestructure**
RSS feed for recent packages. Will model recent releases after this one in a separate PR once we're happy with it.

### CODE REVIEW

**Comment by daveverwer** [comment]
I haven't reviewed this yet, but let's make the autodiscovery meta tags part of this PR too? I also don't see a route for the feeds? Are they exposed yet?

**Comment by finestructure** [comment]
This is WIP - just an easy view for me to see the PR diffs :)

File: Sources/App/Views/PublicPage.swift [hunk 1]
```diff
@@ -31,6 +31,13 @@ class PublicPage {
                 .rel(.stylesheet),
                 .href("/stylesheets/main.min.css?\(resourceReloadQueryString())")
             ),
+            .link(
+                .rel(.alternate),
+                .type("application/rss+xml"),
+                // TODO: Upstream to Plot: .title("...") not allowed here
+                .attribute(named: "title", value: "Swift Package Index – Packages"),
```
[the remark this answers is not in the document]
Comment (finestructure) [reply Sources/App/Views/PublicPage.swift:38]: Resolved in follow-up PR

File: Sources/App/Views/PublicPage.swift [hunk 2]
```diff
@@ -31,6 +31,13 @@ class PublicPage {
                 .rel(.stylesheet),
                 .href("/stylesheets/main.min.css?\(resourceReloadQueryString())")
             ),
+            .link(
+                .rel(.alternate),
+                .type("application/rss+xml"),
+                // TODO: Upstream to Plot: .title("...") not allowed here
```
[the remark this answers is not in the document]
Comment (finestructure) [reply Sources/App/Views/PublicPage.swift:37]: Resolved in follow-up PR

File: Sources/App/Core/RSS.swift [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+
+extension RSSFeed {
+    static func recentPackages(on database: Database, maxItemCount: Int = 100) -> EventLoopFuture<Self> {
+        RecentPackage.fetch(on: database, limit: maxItemCount)
+            .mapEach(RSSFeed.Item.init)
+            .map {
+                RSSFeed(title: "Swift Package Index – Recently Added",
+                        description: "List of recently added Swift packages",
+                        link: URL(string: "feed")!,  // FIXME
```
Comment (finestructure) [inline Sources/App/Core/RSS.swift:56]: Resolved in follow-up PR

**Pull request merged**